.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+# Configuration file for pre-commit (https://pre-commit.com/).
+# Please run `pre-commit run --all-files` when adding or changing entries.
+
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.1.0
+    hooks:
+      - id: codespell
+        args: [--ignore-words=.codespellignore]
+        types_or: [jupyter, markdown, python, shell]
+  - repo: https://github.com/PyCQA/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.950
```

Comment:
Alternatively, or in addition, to my other comment regarding `additional_dependencies` below, change `rev` to the latest version: `v0.991`